src/Hint/Bracket.hs

```diff
@@ -190,10 +190,14 @@ bracket pretty isPartialAtom root = f Nothing
     -- 'x' actually need bracketing in this context?
     f (Just (i, o, gen)) v@(remParens' -> Just x)
       | not $ needBracket i o x, not $ isPartialAtom (Just o) x =
-          rawIdea Suggestion msg (getLoc v) (pretty o) (Just (pretty (gen x))) [] [r] : g x
+          rawIdea Suggestion msg (getLoc v) from to [] [r] : g x
       where
         typ = findType v
         r = Replace typ (toSS v) [("x", toSS x)] "x"
+        (from, to) = reduceJunks (pretty o) (Just (pretty (gen x)))
+        reduceJunks :: String -> Maybe String -> (String, Maybe String)
```

Comment:
Pattern match is not exhaustive. On line 193, you should change `to` to `Just to` so that `reduceJunks` become `String -> String -> (String, String)`.